### ISSUE TITLE
Support Git configuration special expansion variables

### ISSUE THREAD

**Issue by mjcheetham** [microsoft/Git-Credential-Manager-Core#425]
The `git config` command supports an extra `--type=path` argument that also expands special tokens/placeholders in file path values.

From the manpage:

```text
--type <type>
           git config will ensure that any input or output is valid under the given type constraint(s), and
           will canonicalize outgoing values in <type>'s canonical form.
...
           o   path: canonicalize by adding a leading ~ to the value of $HOME and ~user to the home
               directory for the specified user. This specifier has no effect when setting the value (but
               you can use git config section.variable ~/ from the command line to let your shell do the
               expansion.)
...
```

Additionally there appears to be a `%(prefix)` placeholder that expands to the current Git installation prefix.

**Comment by vtbassmatt** [comment]
@vdye is this fixed in #439?

**Comment by vdye** [comment]
Yes! I completely missed this issue in the pull request, but it should be closed.
